Tests/UnitTests/TestObjects/TestSystem.swift

```diff
@@ -36,12 +38,19 @@ final class TestSystem: SystemType {
     // SystemType
     var fileManager: FileManagerType
     var calendar: CalendarType
-    var adIdManager: ASIdentifierManagerType
     var device: UIDeviceType
     var screen: UIScreenType
     var locale: LocaleType
     var bundle: BundleType
-
+    
+    var advertisingId: String? {
```

Comment:
best not to reimplement the actual logic here. You could add a property to the test class here to configure what you want this to return for the class you're testing.